### PR TITLE
GitHub actions - multiarch

### PR DESCRIPTION
**Pull request by baum**
Based on [buildah](https://buildah.io/) and Linux [binfmt](https://github.com/multiarch/qemu-user-static) emulation

- multiarch build with a manifest
- depends on https://github.com/noobaa/noobaa-core/pull/7028

Signed-off-by: Alexander Indenbaum <aindenba@redhat.com>

### CODE REVIEW

**Comment by nimrod-becker** [comment]
Testing some actions issues , closing for now

**Comment by liranmauda** [comment]
@nimrod-becker Can you open a new branch for testing this PR?

**Comment by nimrod-becker** [comment]
Sure, do you want me to create an upstream branch and we can merge this to that branch and then merge the branch once stable to master?

**Comment by baum** [comment]
The new action has to be tested. Could anyone trigger a controlled test run of the new manual build?

Best regards 🖖

File: .github/workflows/manual-full-build.yaml [hunk 1]
```diff
@@ -46,41 +65,65 @@ jobs:
...
+          buildah build \
+            -f src/deploy/NVA_build/NooBaa.Dockerfile \
+            --build-arg GIT_COMMIT=$GIT_COMMIT \
+            --platform=$PLATFORMS \
+            --manifest localhost/noobaa
+          echo "ℹ️ Inspect noobaa manifest"
+          skopeo inspect --raw containers-storage:localhost/noobaa
 
-      - name: Build & Push Docker Images to DockerHub
+      - name: Push Docker Images to DockerHub
```
Comment (dannyzaken) [inline .github/workflows/manual-full-build.yaml:101]: @baum let's remove the push steps. for now we can use this action as a test for our build process and make sure that arm builds are not broken. later on, we can use it to publish multiarch builds, after we have testing pipeline for that

**Review by dannyzaken — CHANGES_REQUESTED** [review]
Currently, it is changing the manual-full-build action. let's change it to a separate action

**Comment by jackyalbo** [comment]
@baum this PR isn't progressing for a while - maybe we should move to draft

**Pull request closed without merging**